### PR TITLE
Update ami_analyze to always extract an 80x80 subarray from full-fram…

### PR DESCRIPTION
**Pull request by dmggh**
…e data. This is equivalent to the user using the SUB80 subarray in the first place, so the subarray matches the location and size of the SUB80 subarray. Updated docs. This addresses github issue #1837.

### CODE REVIEW

File: docs/jwst/ami/main.rst [hunk 1]
```diff
@@ -71,7 +71,12 @@ The ``ami_analyze`` step applies the Lacour-Greenbaum (LG) image plane
 modeling algorithm to a NIRISS AMI image.
 The routine computes a number of parameters, including a model fit (and
 residuals) to the image, fringe amplitudes and phases, and closure phases
-and amplitudes.
+and amplitudes.  For a full-frame exposure, instead of using the FULL subarray,
```
Comment (hbushouse) [inline docs/jwst/ami/main.rst]: I would say "instead of using the FULL **image** ..."

File: docs/jwst/ami/main.rst [hunk 2]
```diff
@@ -71,7 +71,12 @@ The ``ami_analyze`` step applies the Lacour-Greenbaum (LG) image plane
...
+a subarray will be extracted from its center to keep the execution time acceptable.
```
Comment (hbushouse) [inline docs/jwst/ami/main.rst]: Is the subarray actually at (or near) the center of the full image? If not, I would remove the words "from its center".

File: jwst/ami/ami_average.py [hunk 3]
```diff
@@ -1,8 +1,7 @@
-from __future__ import division
-
 #
 #  Module for averaging LG results for a set of AMI exposures
 #
+from __future__ import division
```
Comment (hbushouse) [inline jwst/ami/ami_average.py]: Now that we're only support py3.x environments, we can remove all the `from __future__ import ...` statements, right?

File: jwst/ami/ami_normalize.py [hunk 4]
```diff
@@ -1,12 +1,10 @@
-from __future__ import division
-
 #
 #  Module for normalizing the LG results for a science target by
 #  the LG results for a reference target
 #
+from __future__ import division
```
Comment (hbushouse) [inline jwst/ami/ami_normalize.py]: Remove?

**Comment by dmggh** [comment]
Unfathomable messages from github when I try to resolve this conflict, so will close this PR and try again,

**Pull request closed without merging**